templates/Xamarin/Features/TelemetryHockeyApp.Prism/Param_ProjectName/Services/TrackingService.cs

```diff
@@ -0,0 +1,295 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xamarin.Forms;
+
+namespace Param_ItemNamespace.Services
+{
+    public class TrackingService : ITrackingService
+    {
+        public TrackingLoggingLevel AppLoggingLevel { get; set; } = TrackingLoggingLevel.Information;
+        public TrackingLoggingLevel TrackEventLogLevel { get; set; } = TrackingLoggingLevel.Information;
+        public TrackingLoggingLevel TrackDependencyLogLevel { get; set; } = TrackingLoggingLevel.Information;
+        public TrackingLoggingLevel TrackPageViewLogLevel { get; set; } = TrackingLoggingLevel.Information;
+        public TrackingLoggingLevel TrackExceptionLogLevel { get; set; } = TrackingLoggingLevel.Error;
+        public TrackingLoggingLevel TrackMetricLogLevel { get; set; } = TrackingLoggingLevel.Warning;
+        public int AggregateMetricLoggingDelay { get; set; } = 60000;
+        protected readonly Dictionary<string, TrackingServiceMetric> TrackingServiceMetricCollection = new Dictionary<string, TrackingServiceMetric>();
```

Comment:
Might be able to simplify thread-safe access by switching to ConcurrentDictionary.  I don't see a problem with the current implementation though.

Reply:
Ok